common/transport/mqtt/devdoc/mqtt_base_requirements.md

```diff
@@ -46,6 +46,8 @@ The `connect` method establishes a connection with the server using the config o
 
 **SRS_NODE_COMMON_MQTT_BASE_18_002: [** The `connect` method shall set the `wsOptions.agent` option based on the `mqtt.webSocketAgent` object passed in the `options` structure via the `setOptions` function. **]**
 
+**SRS_NODE_COMMON_MQTT_BASE_18_002: [** The `connect` method shall set the `wsOptions.agent` option based on the `mqtt.webSocketAgent` object passed in the `options` structure via the `setOptions` function. **]**
+
```

Comment:
This doesn't seem to have anything to do with ping timers.

Reply:
good catch. shouldn't look like that.